src/core/fonts.js

```diff
@@ -722,6 +722,11 @@ var Font = (function FontClosure() {
   function int16(b0, b1) {
     return (b0 << 8) + b1;
   }
+  
+  function signed_int16(b0, b1) {
+    var value = (b0 << 8) + b1;
+    return value & (1<<15) ? value - 0x10000 : value;
```

Comment:
Sorry, but there's still missing spaces around `<<`, it should read `(1 << 15)`.